### PR TITLE
doc: improve integration test documentation

### PR DESCRIPTION
**Pull request by limitedmage**

### CODE REVIEW

**Comment by felixfbecker** [comment]
This file was actually intended to be deleted with the new docs. It's somewhat outdated on other fronts too and the docs are more exhaustive. Could we delete the file, or make it _only_ link to the docs?

**Comment by limitedmage** [comment]
@felixfbecker Modified the Readme to only link to the docs, let me know if this makes sense.

File: shared/src/testing/integration/README.md [hunk 1]
```diff
@@ -1,19 +1,5 @@
 # Integration tests
 
-This file contains integration tests for the Sourcegraph UI.
+This folder contains integration tests for the Sourcegraph webapp.
```
Comment (felixfbecker) [inline shared/src/testing/integration/README.md:3]: **nitpick**: This sentence is also one of the outdated sentences - this folder now lives in `shared/`, which just contains shared test helpers for integration tests (both for webapp and browser extension) but no actual tests

**Comment by felixfbecker** [comment]
@limitedmage thanks! 🙌

File: shared/src/testing/integration/README.md (hunk 1 above)
Comment (limitedmage) [reply shared/src/testing/integration/README.md]: Thanks, fixed it!

**Comment by felixfbecker** [comment]
Looks great!

**Pull request merged**